rpm/qmf-qt5.spec

```diff
@@ -33,33 +33,32 @@ Requires:       buteo-syncfw-qt5 >= 0.7.16
 
 Patch1:  0001-fix-tests-installation-path.patch
 Patch2:  0002-Accounts-qt-integration.patch
-Patch3:  0003-Use-a-specific-retry-delay-for-IMAP-idle-connections.patch
-Patch4:  0004-Don-t-save-settings-during-IMAP-logging-in.patch
-Patch5:  0005-Start-messageserver-on-system-startup-in-case-there-.patch
-Patch6:  0006-Add-keepalive-timer-to-IMAP-IDLE-service.patch
-Patch7:  0007-Use-Qt5-booster-to-save-memory.patch
-Patch8:  0008-Listen-to-sync-schedule-changes-from-buteo-sync-fram.patch
-Patch9:  0009-Prevent-push-enabled-status-to-go-out-of-sync.patch
-Patch10:  0010-Add-signature-settings-in-account.patch
-Patch11:  0011-Use-EightBit-encoding-instead-of-Base64-for-text-typ.patch
-Patch12:  0012-Retrieve-message-lists-based-on-the-folder-sync-poli.patch
-Patch13:  0013-Apply-folder-policy-to-always-on-connection.patch
-Patch14:  0014-Allow-a-service-provided-folder-to-be-set-as-the-sta.patch
-Patch15:  0015-Use-a-queued-connection-to-handle-accountsUpdated-si.patch
-Patch16:  0016-Adjust-for-Qt-5.6.patch
-Patch17:  0017-Revert-Fix-bundled-zlib-detection.patch
-Patch18:  0018-Revert-Use-QRandomGenerator-instead-of-qrand.patch
-Patch19:  0019-Revert-Use-range-constructors-for-lists-and-sets.patch
-Patch20:  0020-Revert-Adjust-to-Qt6-QMetaType-API-changes.patch
-Patch21:  0021-Revert-Replace-deprecated-QString-SplitBehavior.patch
-Patch22:  0022-Revert-Fix-disappearance-of-QDateTime-QDate.patch
-Patch23:  0023-Revert-core5compat-addition.patch
-Patch24:  0024-Adjust-qmflist-for-missing-bits-in-5.6.patch
-Patch25:  0025-Revert-loadRelax.patch
-Patch26:  0026-Revert-Set-PLUGIN_CLASS_NAME-in-plugin-.pro-files.patch
-Patch27:  0027-Revert-Bump-version-to-6.0.0-since-we-build-against-.patch
-Patch30:  0028-Fallback-to-sso-credential-plugin.patch
-Patch31:  0029-Revert-private-header-paths.patch
+Patch3:  0003-Don-t-save-settings-during-IMAP-logging-in.patch
+Patch4:  0004-Start-messageserver-on-system-startup-in-case-there-.patch
+Patch5:  0005-Add-keepalive-timer-to-IMAP-IDLE-service.patch
+Patch6:  0006-Use-Qt5-booster-to-save-memory.patch
+Patch7:  0007-Listen-to-sync-schedule-changes-from-buteo-sync-fram.patch
+Patch8:  0008-Prevent-push-enabled-status-to-go-out-of-sync.patch
+Patch9:  0009-Add-signature-settings-in-account.patch
+Patch10:  0010-Use-EightBit-encoding-instead-of-Base64-for-text-typ.patch
+Patch11:  0011-Retrieve-message-lists-based-on-the-folder-sync-poli.patch
+Patch12:  001-Apply-folder-policy-to-always-on-connection.patch
+Patch13:  00134-Allow-a-service-provided-folder-to-be-set-as-the-sta.patch
```

Comment:
Ehm, these got broken. I'll fix on master.

Reply:
Arf, sorry for these bad renaming…

Reply:
On some .specs we have a comment for small script that  can be used to generate the patch snippet. Could be enough scripts here to warrant including a bit help. E.g.

`# i=1; for j in 00*patch; do printf "Patch%04d: %s\n" $i $j; i=$((i+1));done`

Reply:
Indeed, good idea to generate these lines when necessary.